### PR TITLE
[IMP][16.0] fieldservice: improved views concerning categories & tags

### PR DESCRIPTION
**Pull request by NICO-SOLUTIONS**

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

File: fieldservice/views/fsm_location.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
                 <field name="complete_name" />
                 <field
                     name="category_id"
+                    string="Partner-Tags"
```
Comment (hparfr) [inline fieldservice/views/fsm_location.xml]: question: why this special handling for this field ?

there is other fields (street, mobile, etc.) from the partner too.  Specify "partner" only on this field can create confusion for other fields

**Review by hparfr — CHANGES_REQUESTED** [review]
Thanks for these improvments.

I have remarks about Partner-Tags.
I'm ok with the rest of the proposed changes.

File: fieldservice/views/fsm_location.xml (hunk 1 above)
Comment (NICO-SOLUTIONS) [reply fieldservice/views/fsm_location.xml]: i added this string in order to separate it more clearly from the category_id that is used for Worker Categories (Skills).
If there is no clear need, i will remove it. What do you think? @hparfr

Comment (hparfr) [reply fieldservice/views/fsm_location.xml]: Yes please

Comment (NICO-SOLUTIONS) [reply fieldservice/views/fsm_location.xml]: done

**Review by hparfr — APPROVED** [review]
Thanks @NICO-SOLUTIONS

**Pull request merged**